config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -299,7 +299,7 @@ private Token pullComment(int firstChar) {
         // chars JSON allows to be part of a number
         static final String numberChars = "0123456789eE+-.";
         // chars that stop an unquoted string
-        static final String notInUnquotedText = "$\"{}[]:=,+#`^?!@*&\\";
+        static final String notInUnquotedText = "$\"{}()[]:=,+#`^?!@*&\\";
```

Comment:
Reviewing the structure of the distribution of logic across the tokeniser and the doc parser etc its not clear how () may be both as tokens at the same time have _(hello)_ be a single unquoted string.

I will withdraw the implementation entirely.

Pity really because the pull request I made 2 years ago worked fine but that was before the tokeniser was introduced. :(